### DISCUSSION TITLE
A question about the name origin of "Volar"

### DISCUSSION THREAD

**Discussion by favna** [volarjs/volar.js#121] [Q&A] [answered]
This is quite off-topic to coding in and of itself but I was recently talking to a Spanish friend and she mentioned that "Volar" is the Spanish word for the verb "fly". Something clicked in our developer brains and we started to wonder if the project Volar was named after this same verb, just like how Vite is the French word for "fast". 

I can't find an answer to this on the website, repository, or GH Org so I figured I'd try asking through a discussion post.

TL;DR: is Volar's name origin the Spanish translation of "fly"?

**Comment by johnsoncodehk** [comment, marked as answer]
Sorry for the delayed response! "Volar" is one of my favorite songs (https://youtu.be/67a_sfCeZW8), and its name comes from the name of a nightclub in Hong Kong. And yes, its meaning should be "Fly".
